src/dash-adapter.ts

```diff
@@ -206,6 +206,14 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
    * @private
    */
   private _manifestParser: DashManifestParser | null | undefined;
+
+  /**
+   * handel error 1002
+   * @type number
+   * @private
+   */
+  private _errorFirstOccuredTime = 0;
```

Comment:
if you call the other variable error1002X then call this one error1002X too so it would be clear which error it refers to

or you can create a general map of
error code => { count, lastOccuredTime } or something like that